src/App.tsx

```diff
@@ -17,18 +66,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todoStatus={todoStatus}
+                selectStatus={setTodoStatus}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
-            <div className="block">
-              <Loader />
-              <TodoList />
-            </div>
+            {!todos.length
```

Comment:
will be better to handle loader with separate boolean variable
for now if you receive an empty array from server, user will see infinite loader